src/components/DataView/DataViewDisplayGrid.jsx

```diff
@@ -30,9 +34,20 @@ const StyledDiv = styled.div`
 
 	& > .grid {
 		display: grid;
-		grid-template-columns: repeat(5, 1fr);
+		grid-template-columns: repeat(1, 1fr);

```

Comment:
Okay so I've been playing a lot with the sizes and I think we should add 2 more because the jump between 4 columns to 2 feels odd. After researching common breakpoints I think it's best if we add 1 every 300px approx, so it will look something like this:
![image](https://user-images.githubusercontent.com/23409284/176785411-b7d54a48-197c-4fee-a96a-3cecec4cd5a4.png)
Here our biggest amount of columns will be 6 and will descend back to 1. I think leaving the biggest as 5 might not be the best because that screen size it's difficult to get on laptops.